src/main.js

```diff
@@ -10,6 +10,7 @@ import store from './store'
 
 import 'buefy/lib/buefy.css'
 import 'begeta'
+import './assets/css/main.css'
```

Comment:
[BUG] Module not found.

Reply:
I have forgot to add the file.

Could you add the code below to fix color of html when page was stretching?

` html { background-color: black; }`